lib/quicksilver-config-client.js

```diff
@@ -26,6 +26,11 @@ class QuicksilverConfigClient {
     // Parse options.
     this.qsConfigPath = opts.qsConfigPath;
 
+    if (!this.configFileFound()) {
```

Comment:
This won't work. Don't forget, this is a promise. You need to call 'then()' on it.